flocker/node/test/test_deploy.py

```diff
@@ -1050,7 +1055,6 @@ def test_discover_locally_owned_volume_with_size(self):
                     manifestation=Manifestation(
                         dataset=Dataset(
                             dataset_id=DATASET_ID,
-                            metadata=pmap({"name": unit1.name}),
```

Comment:
This is the legacy testcase that is just there temporarily, so not going to be maintaining it.